libs/grout/src/serialization.ts

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
```

Comment:
This duplicates a lot of code that is used for the worker pool data transfer. Worth unifying?

Reply:
Ooooh didn't see that. Yeah it's quite similar! I'm inclined to wait and see what happens with the worker pool code before investing in it, since it might change due to the perf work you're starting. What do you think?

I was thinking about how Grout might be used in a server->worker context as I was building it, and I don't think there's anything that would prevent it from working. Though also not sure how much benefit it would bring.

Reply:
Fine to leave it for now.